src/data-mgt-service/.env

```diff
@@ -0,0 +1,2 @@
+FLASK_APP = app.py
```

Comment:
Could there be a reason why the environmental variables inside the data management service were modified in a PR for the ml service?